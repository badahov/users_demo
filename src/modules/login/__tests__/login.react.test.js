import React from 'react';
import LoginInbox from '../main';

import { configure } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Login', () => {
  it('Layout Index correctly', () => {
    const tree = renderer.create(<LoginInbox />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
