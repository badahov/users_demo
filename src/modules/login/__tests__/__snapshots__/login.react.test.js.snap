// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Layout Index correctly 1`] = `
<section
  className="auth ant-layout"
>
  <main
    className="ant-layout-content"
  >
    <div
      className="ant-row form-box animated fadeInDown"
    >
      <div
        className="ant-col ant-col-10 ant-col-offset-7"
      >
        <h1
          className="logo-name"
        >
          USER
        </h1>
        <h3>
          Добро пожаловать
        </h3>
      </div>
      <div
        className="ant-col ant-col-10 ant-col-offset-7"
      >
        <form
          className="ant-form ant-form-horizontal"
          onSubmit={[Function]}
        >
          <div
            className="ant-row ant-form-item"
          >
            <div
              className="ant-col ant-form-item-control-wrapper ant-col-xs-24 ant-col-sm-24 ant-col-md-14 ant-col-md-offset-5 ant-col-lg-10 ant-col-lg-offset-7"
            >
              <div
                className="ant-form-item-control"
              >
                <span
                  className="ant-form-item-children"
                >
                  <input
                    className="ant-input"
                    data-__field={
                      Object {
                        "name": "login",
                      }
                    }
                    data-__meta={
                      Object {
                        "name": "login",
                        "originalProps": Object {
                          "disabled": false,
                          "placeholder": "Логин",
                          "type": "text",
                        },
                        "ref": null,
                        "rules": Array [
                          Object {
                            "message": "Впишите логин",
                            "required": true,
                            "whitespace": true,
                          },
                        ],
                        "trigger": "onChange",
                        "validate": Array [
                          Object {
                            "rules": Array [
                              Object {
                                "message": "Впишите логин",
                                "required": true,
                                "whitespace": true,
                              },
                            ],
                            "trigger": Array [
                              "onChange",
                            ],
                          },
                        ],
                        "valuePropName": "value",
                      }
                    }
                    disabled={false}
                    id="register_login"
                    onChange={[Function]}
                    onKeyDown={[Function]}
                    placeholder="Логин"
                    type="text"
                    value=""
                  />
                </span>
              </div>
            </div>
          </div>
          <div
            className="ant-row ant-form-item"
          >
            <div
              className="ant-col ant-form-item-control-wrapper ant-col-xs-24 ant-col-sm-24 ant-col-md-14 ant-col-md-offset-5 ant-col-lg-10 ant-col-lg-offset-7"
            >
              <div
                className="ant-form-item-control"
              >
                <span
                  className="ant-form-item-children"
                >
                  <span
                    className="ant-input-password ant-input-affix-wrapper"
                  >
                    <input
                      action="click"
                      className="ant-input"
                      data-__field={
                        Object {
                          "name": "password",
                        }
                      }
                      data-__meta={
                        Object {
                          "name": "password",
                          "originalProps": Object {
                            "action": "click",
                            "inputPrefixCls": "ant-input",
                            "placeholder": "Пароль",
                            "prefixCls": "ant-input-password",
                            "visibilityToggle": true,
                          },
                          "ref": null,
                          "rules": Array [
                            Object {
                              "message": "Впишите пароль",
                              "required": true,
                            },
                            Object {
                              "validator": [Function],
                            },
                          ],
                          "trigger": "onChange",
                          "validate": Array [
                            Object {
                              "rules": Array [
                                Object {
                                  "message": "Впишите пароль",
                                  "required": true,
                                },
                                Object {
                                  "validator": [Function],
                                },
                              ],
                              "trigger": Array [
                                "onChange",
                              ],
                            },
                          ],
                          "valuePropName": "value",
                        }
                      }
                      disabled={false}
                      id="register_password"
                      onChange={[Function]}
                      onKeyDown={[Function]}
                      placeholder="Пароль"
                      style={null}
                      type="password"
                      value=""
                    />
                    <span
                      className="ant-input-suffix"
                    >
                      <i
                        aria-label="icon: eye-invisible"
                        className="anticon anticon-eye-invisible ant-input-password-icon"
                        onClick={[Function]}
                        onMouseDown={[Function]}
                        tabIndex={-1}
                      >
                        <svg
                          aria-hidden="true"
                          className=""
                          data-icon="eye-invisible"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 0 0 0-51.5zm-63.57-320.64L836 122.88a8 8 0 0 0-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 0 0 0 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 0 0 0 11.31L155.17 889a8 8 0 0 0 11.31 0l712.15-712.12a8 8 0 0 0 0-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 0 0-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 0 1 146.2-106.69L401.31 546.2A112 112 0 0 1 396 512z"
                          />
                          <path
                            d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 0 0 227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 0 1-112 112z"
                          />
                        </svg>
                      </i>
                    </span>
                  </span>
                </span>
              </div>
            </div>
          </div>
          <div
            className="ant-row ant-form-item"
          >
            <div
              className="ant-col ant-form-item-control-wrapper ant-col-xs-24 ant-col-xs-offset-0 ant-col-sm-24 ant-col-sm-offset-0"
            >
              <div
                className="ant-form-item-control"
              >
                <span
                  className="ant-form-item-children"
                >
                  <button
                    className="ant-btn ant-btn-primary"
                    onClick={[Function]}
                    type="submit"
                  >
                    <span>
                      Войти
                    </span>
                  </button>
                </span>
              </div>
            </div>
          </div>
        </form>
      </div>
    </div>
  </main>
</section>
`;
